utils/instructions/Dual/index.ts

```diff
@@ -0,0 +1,219 @@
+      connection.current,
+      form.quoteTreasury.pubkey
+    )
+    const quoteMint = quoteTreasuryAccount?.account.mint
+
+    if (!baseMint || !quoteMint) {
+      return {
+        serializedInstruction,
+        isValid: false,
+        governance: form.baseTreasury?.governance,
+        additionalSerializedInstructions: [],
+        chunkSplitByDefault: true,
+        chunkBy: 1,
+      }
+    }
+
+    const configInstruction = await so.createConfigInstruction(
+      form.optionExpirationUnixSeconds,
+      form.optionExpirationUnixSeconds,
+      form.numTokens,
+      form.lotSize,
+      form.soName,
+      //use sol wallet as authority
+      form.payer.extensions.transferAddress!,
+      baseMint,
+      //use helper account as base account
+      helperTokenAccount.publicKey,
+      quoteMint,
+      form.quoteTreasury.pubkey
+    )
+
+    additionalSerializedInstructions.push(
+      serializeInstructionToBase64(configInstruction)
+    )
+
+    const initStrikeInstruction = await so.createInitStrikeInstruction(
+      form.strike,
+      form.soName,
+      //authority sol wallet
+      form.payer.extensions.transferAddress!,
+      baseMint
+    )
+    additionalSerializedInstructions.push(
+      serializeInstructionToBase64(initStrikeInstruction)
+    )
+
+    const soMint = await so.soMint(form.strike, form.soName, baseMint)
+    const userSoAccount = await findAssociatedTokenAddress(
+      new PublicKey(form.userPk),
+      soMint
+    )
+
+    if (!(await connection.current.getAccountInfo(userSoAccount))) {
+      const [ataIx] = await createAssociatedTokenAccount(
+        wallet.publicKey,
+        new PublicKey(form.userPk),
+        soMint
+      )
+      additionalSerializedInstructions.push(serializeInstructionToBase64(ataIx))
+    }
+
+    const issueInstruction = await so.createIssueInstruction(
+      form.numTokens,
+      form.strike,
+      form.soName,
+      //authority sol wallet
+      form.payer.extensions.transferAddress!,
+      baseMint,
+      userSoAccount
+    )
+
+    additionalSerializedInstructions.push(
+      serializeInstructionToBase64(issueInstruction)
+    )
+
+    //after everything we close helper account
+    additionalSerializedInstructions.push(
+      serializeInstructionToBase64(
+        closeAccount({
+          source: helperTokenAccount.publicKey,
+          //sol wallet
+          destination: form.payer.extensions.transferAddress,
+          //owner governance or sol wallet same as baseTokenAccount
+          owner: form.baseTreasury.extensions.token?.account.owner,
+        })
+      )
+    )
+
+    const obj: UiInstruction = {
+      serializedInstruction,
+      isValid: true,
+      prerequisiteInstructions: prerequisiteInstructions,
+      prerequisiteInstructionsSigners: [helperTokenAccount],
+      governance: form.baseTreasury?.governance,
+      additionalSerializedInstructions,
+      chunkSplitByDefault: true,
+      chunkBy: 2,
+    }
+    return obj
```

Comment:
Nit, no need to use an intermediary variable to type the object. The returned value of the function `getConfigInstruction` is typed:

```suggestion
    return  {
      serializedInstruction,
      isValid: true,
      prerequisiteInstructions: prerequisiteInstructions,
      prerequisiteInstructionsSigners: [helperTokenAccount],
      governance: form.baseTreasury?.governance,
      additionalSerializedInstructions,
      chunkSplitByDefault: true,
      chunkBy: 2,
    }
```